priv/repo/migrations/20220828030827_create_authentication_tables.exs

```diff
@@ -0,0 +1,16 @@
+defmodule Hexagon.Repo.Migrations.CreateAuthenticationTables do
+  use Ecto.Migration
+
+  def change do
+    create table(:authentication_keys) do
+      add :name, :string, null: false
+      add :token, :binary, null: false, size: 64
```

Comment:
I think you could do a very basic permissions with push (bool) and pull (bool).

Reply:
Yeah. I think this is what I've though of so far

- push and pull bools
- admin, read, write enum
- string list of scopes

The last one is what hex uses, but I can probably translate on the controller to make it work with any of these.